app/cinema/bar.py

```diff
@@ -0,0 +1,7 @@
+from app.people.customer import Customer
+
+
+class CinemaBar:
+    @staticmethod
+    def sell_product(product: str, customer: Customer) -> str:

```

Comment:
Invalid type hint for return type
```suggestion
    def sell_product(product: str, customer: Customer) -> None:
```

Reply:
Done